recipes/betse/meta.yaml

```diff
@@ -0,0 +1,84 @@
+{% set name = "betse" %}
+{% set version = "0.7.0" %}
+{% set sha256 = "845896f4c30b3970d5bd9501e0ca754eef5af0d742de76103f66c8c8c58e4eca" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  entry_points:
+    - betse = betse.__main__:main
+  script: python -m pip install --no-deps --ignore-installed .
+  noarch: python
+
+requirements:
+  build:
+    - pip
+    - python >=3.4
+    - dill >=0.2.3
+    - graphviz >=2.38.0
+    - matplotlib >=1.5.0
+    - networkx >=2.0
+    - numpy >=1.8.2
+    - pillow >=2.3.0
+    - pydot >=1.0.28
+    - pyyaml >=3.10
+    - scipy >=0.12.0
+    - setuptools >=3.3
+    - six >=1.5.2
+    - ffmpeg
+    - pyside2
```

Comment:
Would chop this list down to what is strictly needed for the build phase. So `python`, `pip`, `setuptools`. Guessing nothing else here is needed at build time.